src/coreclr/inc/corinfo.h

```diff
@@ -1227,6 +1227,7 @@ enum CORINFO_RUNTIME_LOOKUP_KIND
 struct CORINFO_LOOKUP_KIND
 {
     bool                        needsRuntimeLookup;
+    bool                        inlinedLookup;
```

Comment:
I do not understand why this new flag is needed. What is not going to work without it?

Reply:
@jkotas my initial version didn't need it at all, I added it for additional asserts in JIT, I think my final version won't have it